main.tf

```diff
@@ -0,0 +1,142 @@
+locals {
+  resource_prefix = "naveh-tf-final"
+}
+
+resource "azurerm_resource_group" "hub" {
+  location = var.location
+  name     = "${local.resource_prefix}-hub"
+}
+
+resource "azurerm_resource_group" "spoke" {
+  location = var.location
+  name     = "${local.resource_prefix}-spoke"
+}
+
+module "hub_virtual_network" {
```

Comment:
we prefer to get all the subnets as one big object and not separating it to different blocks